src/block/accordion/frontend-accordion.js

```diff
@@ -30,6 +30,10 @@ class StackableAccordion {
 					el.doAnimate = false
 					const preHeight = el.dataset.preHeight
 
+					// Prevent text selection while animating
+					const content = el.querySelector( '.stk-block-accordion__content' )
+					content.style.userSelect = 'none'
```

Comment:
It seems that doing this on the content would still make the text highlight. I would recommend doing this on the element itself to prevent this:
```suggestion
					// Prevent text selection while animating
					el.style.userSelect = 'none'
```